oversight/root_urls.py

```diff
@@ -1,11 +1,14 @@
-from django.conf.urls import patterns, include, url
+from django.conf.urls import include, url
 from django.views.generic.base import RedirectView
 
+from django.conf import settings
+from django.conf.urls.static import static
+
 from django.contrib import admin
 
 
-urlpatterns = patterns('',
+urlpatterns = [
     url(r'^oversight/', include('oversight.urls')),
-    url(r'^admin/', include(admin.site.urls)),
+    url(r'^admin/', admin.site.urls),
     url(r'^$', RedirectView.as_view(pattern_name='oversight_index')),
-)
+]+ static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
```

Comment:
Wovon sind die staticfiles davor ausgeliefert worden?

Reply:
Das Setup am Raspberry verwendet für static files nginx. Die Zeile oben war zum Testen/Entwickeln.
Wie würde man das richtig machen?

Reply:
Bei testen & entwicklen liefert Django die Sachen automatisch aus solange du:
 * ./manage.py runserver verwendest
 * staticfiles in INSTALLED_APPS ist